src/main/webapp/user_script.js

```diff
@@ -20,7 +20,9 @@ function loadProfile() {
         bioSection.innerHTML = profile.bio;
         bioSection.value = profile.bio;
         const profileImage = document.getElementById("avatar");
-        profileImage.src = "https://icotar.com/initials/" + profile.username;
+        const username = profile.username;
+        const avatarLetter = username.charAt(0);
```

Comment:
let's add another TODO for edit/create profile to prevent users from submitting usernames that break this assumption. 

e.g. blank usernames, or usernames that start with spaces